sklearn/utils/tests/test_sparsefuncs.py

```diff
@@ -101,12 +102,11 @@ def test_incr_mean_variance_axis():
         X = np.atleast_2d(X)
         X_lil = sp.lil_matrix(X)
         X_csr = sp.csr_matrix(X_lil)
-        assert_raises(TypeError, incr_mean_variance_axis, axis,
-                      last_mean, last_var, last_n)
-        assert_raises(TypeError, incr_mean_variance_axis, axis,
-                      last_mean, last_var, last_n)
-        assert_raises(TypeError, incr_mean_variance_axis, X_lil, axis,
-                      last_mean, last_var, last_n)
+
+        with pytest.raises(TypeError):
```

Comment:
So we ran this test twice on master ;)

Reply:
Indeed, it could have been on purpose if the function was mutating an argument but that is not the case here :)